go/registry/api/api.go

```diff
@@ -201,15 +201,21 @@ type Backend interface {
 
 	// WatchRuntimes returns a stream of Runtime.  Upon subscription,
 	// all runtimes will be sent immediately.
-	WatchRuntimes() (<-chan *Runtime, *pubsub.Subscription)
+	WatchRuntimes(context.Context) (<-chan *Runtime, pubsub.ClosableSubscription, error)
 
-	// ToGenesis returns the genesis state at specified block height.
-	ToGenesis(context.Context, int64) (*Genesis, error)
+	// StateToGenesis returns the genesis state at specified block height.
+	StateToGenesis(context.Context, int64) (*Genesis, error)
 
 	// Cleanup cleans up the registry backend.
 	Cleanup()
 }
 
+// IDQuery is a registry query by ID.
+type IDQuery struct {
```

Comment:
Why not just `Query`?

Reply:
Because there could be other kinds of queries.